index.bs

```diff
@@ -568,6 +568,8 @@ ReadableStream(<var>underlyingSource</var> = {}, { <var>size</var>, <var>highWat
 </div>
 
 <emu-alg>
+  1. If _writable_ is *undefined* or _readable_ is *undefined*, throw a *TypeError* exception indicating that undefined
+     values are not permitted.
```

Comment:
Sorry, I was confused, readable obviously exists. And if `pipeTo` is overridden, it is observably different whether we throw an error for writable or not. Hmm.